editor/editor_file_system.cpp

```diff
@@ -2142,6 +2142,8 @@ EditorFileSystem::EditorFileSystem() {
 	}
 	//this should probably also work on Unix and use the string it returns for FAT32
 	using_fat_32 = da->get_filesystem_type() == "FAT32";
+	//this should probably also work on Unix and use the string it returns for exFAT
+	using_ex_fat = da->get_filesystem_type() == "exFAT";
```

Comment:
Changes are fine, but IMO it's not necessary to have two booleans enabling the same features. So I'd just do
```
using_fat_32 = (da->get_filesystem_type() == "FAT32" || da->get_filesystem_type() == "exFAT");
```

Reply:
Could you amend the commit to do this change?

Reply:
@kpkhxlgy0 Bump :slightly_smiling_face:

Reply:
I force-pushed the requested changes.